mne/viz/topomap.py

```diff
@@ -46,6 +46,9 @@ def _prepare_topo_plot(inst, ch_type, layout):
     for ii, this_ch in enumerate(info['chs']):
         this_ch['ch_name'] = clean_ch_names[ii]
     info['bads'] = _clean_names(info['bads'])
+    for comp in info['comps']:
+        comp['data']['col_names'] = _clean_names(comp['data']['col_names'])
```

Comment:
This will modify the object inplace. Do we know that it has been copied already everywhere `_prepare_topo_plot` is used?

Is there some new test that shows why this is necessary (i.e., fails on master but passes here)?

Reply:
info is a copy see https://github.com/mne-tools/mne-python/blob/master/mne/viz/topomap.py#L37

mne/viz/tests/test_topomap.py::test_ctf_plotting currently fails without this line. 

I can probably eliminate this line if we drop the comp matrices from the info before calling `_prepare_topo_plot`. I imagine this is also a problem from the other topomap plotting functions-  currently only plot_evoked_topomap drops the comp matrices. I'll look at others